.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules/
-development.json
+ideahacks/config/development.json
```

Comment:
Why this change? The previous .gitignore still caught our development file, and we don't have any other files named `development.json`. I understand that specificity is nice for file-naming flexibility, but it's general like this so a new committee member doesn't put the file in the wrong folder, then push it onto GitHub